client/prometheus.go

```diff
@@ -0,0 +1,233 @@
+package client
+
+import (
+	"net/http"
+	"github.com/hellofresh/stats-go/bucket"
+	"github.com/hellofresh/stats-go/timer"
+	"sync"
```

Comment:
use `goimports` sorting with groupping stdlib imports (Go -> Imports in GoLand Preferences)